tiledb/cloud/client.py

```diff
@@ -5,8 +5,9 @@
 from . import config
 from . import tiledb_cloud_error
 from .rest_api import ApiException as GenApiException
-import urllib
 import os
+import typing as t
```

Comment:
Please use mypy style explicit imports.

Reply:
vscode's python extension can add them automatically (for better or sometimes worse)

Reply:
Done.

Reply:
Sorry, I was unclear there -- by explicit imports I meant: `from typing import Union, Optional, ...`. That's the mypy convention and what we've adopted in other repos.

Reply:
ah jeez, I had not been staging my new changes when I ran `git commit --amend` so it was happily pushing no changes and just updating the commit time.

now **acutally** fixed.

Reply:
Thank you!